pkg/server/api/bundle/v1/service.go

```diff
@@ -151,6 +138,43 @@ func (s *Service) AppendBundle(ctx context.Context, req *bundle.AppendBundleRequ
 	return bundle, nil
 }
 
+func parseX509Authorities(certs []*types.X509Certificate) ([]*common.Certificate, error) {
+	var rootCAs []*common.Certificate
+	for _, rootCA := range certs {
+		if _, err := x509.ParseCertificates(rootCA.Asn1); err != nil {
+			return nil, err
+		}
+
+		rootCAs = append(rootCAs, &common.Certificate{
+			DerBytes: rootCA.Asn1,
+		})
+	}
+
+	return rootCAs, nil
+}
+
+func parseJWTAuthorities(keys []*types.JWTKey) ([]*common.PublicKey, error) {
+	var jwtKeys []*common.PublicKey
+	for _, key := range keys {
+		if _, err := x509.ParsePKIXPublicKey(key.PublicKey); err != nil {
+			return nil, err
+		}
+
+		// TODO: what if KeyId is on datastore?
```

Comment:
I'm not sure I understand this TODO. Can you elaborate a little?

Reply:
what happenns if we have a jwtkey on datastore where we have a KeyID that is the same than the bundle we are attaching, it will just be updated? added and have 2 keys with same KeyID?

Reply:
The datastore "append" operation will replace the older key that shares an ID, which is the expected behavior.